content/knowledge-base/install-unity-version.md

```diff
@@ -0,0 +1,117 @@
+---
+description: How to install a different Unity
+title: Installing a different Unity version
+weight: 13
+---
+
+# Installing different Unity versions
+Each build machine image has a specific version of Unity installed. You can find out the specific Unity version by consulting the build machine specification in the documentation. For example, the [macOS build machine specification (Xcode 13.3+ / Unity)](https://docs.codemagic.io/specs/versions-macos-xcode-13-3/#unity-2020331f1) has Unity version version 2020.3.31f1.
+
+If you need to use a different Unity version then it is possible to use the Unity Hub CLI to download and install a different Unity Editor version and target support files for that version. 
+
+License activation and return takes place with the Unity version already installed on the machine, but building of the Xcode project or Android binary will use the version of Unity you install. 
+
+
+## Getting the Unity version number and changeset id
+
+In order to install a different version you will need to know the full Unity version number and its **changeset** id. These details can be found in the Unity download archive [here](https://unity3d.com/get-unity/download/archive). To get the changeset id, first select the version you intend to use and then click on the ‘Release Notes’ button. Scroll to the bottom of the release notes and you will see the changeset id which you should make a note of. 
+
+
+## Environment variables
+
+You will need to a three environment variables in your codeamagic.yaml called  `UNITY_VERSION`, `UNITY_VERSION_CHANGSET`, and `UNITY_VERSION_BIN`. 
+
+You should set the value of `UNITY_VERSION` to the full version number as display on the Unity Hub download archive page. For example, if the version is 2019.4.38f1, set the variable as follows:
+
+`UNITY_VERSION: 2019.4.38f1`
+
+The value of `UNITY_VERSION_CHANGESET` should be set using the **changeset** id that you obtained from the release notes page on the Unity Hub download archive. For example, if the changeset id is fdbb7325fa47 then set the variable as follows:
+
+`UNITY_VERSION_CHANGESET: fdbb7325fa47`
+
+The `UNITY_VERSION_BIN` should be set as follows so the Unity binary path is declared for the version you want to build with:
+
+`UNITY_VERSION_BIN: /Applications/Unity/Hub/Editor/${UNITY_VERSION}/Unity.app/Contents/MacOS/Unity`
+
+
+## Activating Unity
+
+Even though you are installing a different version of Unity to build your apps with, you should use the default Unity version that is already installed on the machine. Unity Hub CLI commands not work correctly if a license is not already active on the machine.
```

Comment:
This section needs to be clarified a bit. ... you should activate your license using the default Unity version already installed on the machine? .. activate the default Unity version?

Reply:
You need to activate the license using the version already installed on the machine. Is this better:


## Activating Unity

Even though you are installing a different version of Unity to build your apps with, you should activate your license using the default Unity version already installed on the machine. Unity Hub CLI commands not work correctly if a license is not already active on the machine.

Reply:
That would work

Reply:
Still, one missing word:
Unity Hub CLI commands **do** not work correctly if a license is not already active on the machine.

Reply:
corrected